src/Periphery/Executor.sol

```diff
@@ -261,6 +262,20 @@ contract Executor is ILiFi, ReentrancyGuard, TransferrableOwnership {
         return balances;
     }
 
+    /// @dev Will be called as part of an ERC721 token transfer to this contract (see IERC721Receiver for more details)
+    /// @param operator The address of the tx initiator
+    /// @param from The address from which the ERC721 token was transferred to this contract
+    /// @param tokenId The ID of the token that was transferred
+    /// @return uint256[] Array of token balances.
```

Comment:
return value is not balances(uint256[]) but selector(bytes4)

Reply:
updated